### ISSUE TITLE
Breaks if config file ends with an extension

### ISSUE THREAD

**Issue by dgudim** [VorpalBlade/chezmoi_modify_manager#14]
## Describe the bug
When adding a config file ending in ```.conf``` for example, when running ```chezmoi apply``` you get something like this: 

```sh
Error: Failed to open source file at: "/home/kloud/.local/share/chezmoi/dot_config/nheko/nheko.conf.src.ini"

Caused by:
    No such file or directory (os error 2)
chezmoi: exit status 1
```
(I tried adding nheko config, it created ```nheko.src.ini``` instead of ```nheko.conf.src.ini```)

## To Reproduce
- Add any config file with an extension

## Expected behavior
- No errors

## Actual behavior
- Does not work, renaming the ini file fixes the problem

## Environment (please complete the following information):
 - OS: Arch Linux
 - Architecture: x86-64
 - Chezmoi version: 2.33.5
 - chezmoi_modify_manager version/commit ID: v2.0.0

### chezmoi doctor output

<details>

```console
RESULT    CHECK                MESSAGE
warning   version              v2.33.5, built at 2023-05-12T01:57:32Z
ok        latest-version       v2.33.5
ok        os-arch              linux/amd64 (EndeavourOS)
ok        uname                Linux sauron 6.2.10-arch1-1 #1 SMP PREEMPT_DYNAMIC Fri, 07 Apr 2023 02:10:43 +0000 x86_64 GNU/Linux
ok        go-version           go1.20.4 (gc)
ok        executable           /usr/bin/chezmoi
ok        config-file          ~/.config/chezmoi/chezmoi.toml, last modified 2023-05-06T15:51:42+03:00
ok        source-dir           ~/.local/share/chezmoi is a git working tree (clean)
ok        suspicious-entries   no suspicious entries
ok        working-tree         ~/.local/share/chezmoi is a git working tree (clean)
ok        dest-dir             ~ is a directory
ok        umask                022
ok        cd-command           found /bin/bash
ok        cd-args              /bin/bash
info      diff-command         not set
ok        edit-command         found /usr/bin/nano
ok        edit-args            /usr/bin/nano
ok        git-command          found /usr/bin/git, version 2.40.1
ok        merge-command        found /usr/bin/vimdiff
ok        shell-command        found /bin/bash
ok        shell-args           /bin/bash
ok        age-command          found /usr/bin/age, version 1.1.1
ok        gpg-command          found /usr/bin/gpg, version 2.2.41
info      pinentry-command     not set
info      1password-command    op not found in $PATH
info      bitwarden-command    bw not found in $PATH
info      dashlane-command     dcli not found in $PATH
info      gopass-command       gopass not found in $PATH
ok        keepassxc-command    found /usr/bin/keepassxc-cli, version 2.7.4
info      keepassxc-db         not set
info      keeper-command       keeper not found in $PATH
info      lastpass-command     lpass not found in $PATH
info      pass-command         pass not found in $PATH
info      passhole-command     ph not found in $PATH
info      rbw-command          rbw not found in $PATH
info      vault-command        vault not found in $PATH
info      secret-command       not set
```

</details>

**Comment by VorpalBlade** [comment]
Is this for adding new configs or for updating existing configs? I ran into and fixed an issue a couple of minutes ago where by re-adding existing files were broken (a relative path was used instead of an absolute one).

Also, when adding the problematic file, what is the exact command you are using?

**Comment by VorpalBlade** [comment]
> Wow, that was fast

I happened to be working on this (*and* just finishing up said work) just as your bug report showed up. Do not expect that in general!

A new release 2.0.2 should be out soon with this fix. I would like to first look into another improvement whereby the GitHub actions also updates the AUR package automatically. In the meantime you could use the -git package, or manually rename the file.

**Comment by dgudim** [comment]
> I happened to be working on this (and just finishing up said work) just as your bug report showed up.

Funny enough, I just installed modify manager, added 1 config, and immediately filed a bug report :)
Nice timings

> In the meantime you could use the -git package, or manually rename the file.

Yeah, I'll just rename for now, thanks

**Comment by VorpalBlade** [comment]
> Yeah, I'll just rename for now, thanks

Note that re-adding an existing file was broken in 2.0.0, I fixed that in 2.0.1, which I published just as this bug report showed up.

**Comment by dgudim** [comment]
Already updated, thanks again
